examples/speech_to_text/counter_in_tgt_lang_agent.py

```diff
@@ -14,14 +14,11 @@ class CounterInTargetLanguage(SpeechToTextAgent):
     def __init__(self, args):
         super().__init__(args)
         self.wait_seconds = args.wait_seconds
-        self.tgt_lang = args.tgt_lang
 
     @staticmethod
     def add_args(parser):
         parser.add_argument("--wait-seconds", default=1, type=int)
-        parser.add_argument(
-            "--tgt-lang", default="en", type=str, choices=["en", "es", "de"]
-        )
+        parser.add_argument("--tgt-lang")
```

Comment:
`--tgt-lang` arg is being added at two places (here and in `simuleval/options.py`) , this is why you were seeng the error during the demo today.

```suggestion
```